app/controllers/admin/facebook_posts_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Admin::FacebookPostsController < ApplicationController
+  def index
+    tweets = Tweet.in_organization(current_organization).not_send_to_facebook
+    authorize tweets
+    render :index, locals: { tweets: tweets }
+  end
+
+  def send_to_facebook
+    SendTweetToFacebook.call(user: current_user, tweet_id: params[:id], organization: current_organization)
```

Comment:
make this call async and updating listing using action cable, when post was posted to facebook (a new idea ✌️)

Reply:
Will be changed in next PR